LiskHQ/lisk-mobile
# Create tasks in a background thread - Closes #299

# What was the bug or feature?
It is described in #299

### How did I fix it?
Please provide a short description of your changes.

## Type of change
- Enhancement (a non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

### How to test it?
1. login to an account.
2. closes the app in the mobile device
3. send some token to the first account.
4. you should receive a notification on Android or IOS after 15 minutes

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
